login/views.py

```diff
@@ -20,9 +21,16 @@ def user_login(request):
                         username=request.data['email'], 
                         email=request.data['email'], 
                         password=request.data['password'])
+
     if not user:
         data['response'] = 'Invalid email or password.'
-        return Response(data=data, status=status.HTTP_401_UNAUTHORIZED)   
+        return Response(data=data, status=status.HTTP_401_UNAUTHORIZED)
+
+    # Filter user based on VerifiedUser
+    is_verified = VerifiedUser.objects.get(user=user).verified
```

Comment:
mungkin ini bisa tambahin case kalau verifieduser nya not exists luck, jadi autoverify aja (ini kepake buat user yg udah existing) wkwkwk
```
try: 
   is_verified = VerifiedUser.objects.get(user=user).verified
except VerifiedUser.DoesNotExists:
   VerifiedUser.objects.create(user=user, verified=True)
   is_verified = True
```

Reply:
Done